nes.mk

```diff
@@ -2,6 +2,8 @@ override CAFLAGS += -g
 LDFLAGS =
 VPATH = build
 
+BASE16 = $(shell command -v basenc 1>/dev/null && echo basenc --base16 || echo basez --base16 -c)
```

Comment:
`basez` doesn't seemed installed out-of-the-box and doesn't seem all that common (Arch doesn't have it, for example). How about using xxd instead? `xxd -c1 -p` to encode and `xxd -r -p` to decode. That means we'd need two different variables here, one for encoding and one for decoding.

xxd would probably have been a better decision in the first place, but I wanted to limit myself to very few dependencies and didn't notice that basenc has only been available since coreutils 8.31 from 2019. It appears xxd is available on the git-for-windows install and is available for OS X. xxd is so common, it is probably fine to just replace basenc and always use xxd.